ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -104,6 +104,66 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-advanced-medium-low-f7
+  cron: 38 17 2,9,16,23,30 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Medium,Low)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-ad
+      REGION: ap-northeast-1
+      RHCS_ENV: staging
+    test:
+    - ref: rhcs-e2e-tests
+    workflow: rhcs-aws-sts
+- as: rosa-sts-private-medium-low-f7
+  cron: 38 2 2,9,16,23,30 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Medium,Low)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-pl
+      REGION: us-east-1
+      RHCS_ENV: staging
+    test:
+    - ref: rhcs-e2e-tests
+    workflow: rhcs-aws-sts
+- as: rosa-sts-advanced-negative-f7
+  cron: 38 6 1,8,15,22,29 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: day1-negative&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-ad
+      REGION: ap-northeast-1
+      RHCS_ENV: staging
+    post:
+    - ref: rhcs-general-teardown
+    pre:
+    - ref: rhcs-general-setup
+    test:
+    - ref: rhcs-general-tests
+    workflow: rhcs-aws-sts
+- as: rosa-sts-private-negative-f7
+  cron: 38 2 2,9,16,23,30 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: day1-negative&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-pl
+      REGION: us-east-1
+      RHCS_ENV: staging
+    post:
+    - ref: rhcs-general-teardown
+    pre:
+    - ref: rhcs-general-setup
+    test:
+    - ref: rhcs-general-tests
+    workflow: rhcs-aws-sts
```

Comment:
And also remove workflow, instead just has test

Reply:
I'll try to run just tests

Reply:
@xueli181114 Can't run just tests part, failing on states archive:
```
+ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+ ls /tmp/secret
+ cp -r /root/terraform-provider-rhcs /alabama/
+ cp -r /alabama/terraform-provider-rhcs/tests/tf-manifests /tmp/secret/tf-manifests
+ cd /tmp/secret
+ tar -xvf statefiles.tar.gz
tar: statefiles.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

Reply:
@xueli181114 Do you think return to the general tests script but without the pre?
In post steps I would probably want to delete possible leftovers (account/operator roles, VPC, etc)

Reply:
@xueli181114 why we don't need workflow? it holds the teardown no? without it the clusters does not
get deleted and stayed in the the staging env.